test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -24,26 +24,65 @@ test('IndexTable - bulk action', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  let isCheckboxCorrect: boolean, tempText: string | null;
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(false)
+  );
+  expect(isCheckboxCorrect).toBe(true);
```

Comment:
can you assert that the bulk action is not visible initially?

Reply:
sure!